composer.json

```diff
@@ -12,15 +12,25 @@
         "emit"
     ],
     "require-dev": {
-        "phpunit/phpunit": "3.7.*"
+        "phpunit/phpunit": "^5.5",
```

Comment:
Doing so causes the problem:
```
  Problem 1
    - phpunit/phpunit 4.8.20 requires php ~5.3.3|~5.4|~5.5|~5.6 -> your PHP version (7.0.13) does not satisfy that requirement.
```

So we can't test the library against php 5.5 and php 7.0 with the same composer.json

I think it's better to remove php 5.5 from .travis.yml, leave composer.json as is.

And then introduce some kludge for testing with php 5.5.